src/js/components/about/AboutContent.jsx

```diff
@@ -64,12 +63,15 @@ const aboutSections = [
     }
 ];
 
-const propTypes = {
-    location: PropTypes.object // Router location object
-};
+const AboutContent = () => {
+    const location = useLocation();
+    const history = useHistory();
+    const useQuery = (() => (
+        new URLSearchParams(location.search)
+    ));
+    const query = useQuery();
```

Comment:
Can we use this guy instead? https://github.com/fedspendingtransparency/usaspending-website/blob/dev/src/js/helpers/queryParams.js#L3

Reply:
Yes, good call!